generator/CPP11/include_v2.0/msgmap.hpp

```diff
@@ -39,7 +39,7 @@ class MsgMap {
 	}
 
 	template<typename _T>
-	void operator<< (const _T data);
+	void operator<< (const _T &data);
```

Comment:
What is the reason to pass by ref?

Reply:
It avoids a copy (though there won't be any benefit for data types <= pointer size).

Reply:
It even can give speed penalty because you have to access pointer.